src/main/java/com/wiley/autotest/selenium/context/OurSearchStrategy.java

```diff
@@ -41,7 +57,7 @@ public OurSearchStrategy nullOnFailure() {
     }
 
     public long getTimeout() {
```

Comment:
@vefimofff I agree with @estuold that you have two constructors the one uses timeout (even if it is autowired) and the second uses customtimeout. If getTimeout returns customtimeout then it would be better to use naming getCustomTimeout.